src/alchemlyb/convergence/convergence.py

```diff
@@ -137,3 +138,166 @@ def forward_backward_convergence(df_list, estimator='MBAR', num=10, **kwargs):
+            1  3.078106  3.078567            0.2
+            2  3.072561  3.047357            0.3
+            3  3.048325  3.057527            0.4
+            4  3.049769  3.037454            0.5
+            5  3.034078  3.040484            0.6
+            6  3.043274  3.032495            0.7
+            7  3.035460  3.036670            0.8
+            8  3.042032  3.046597            0.9
+            9  3.044149  3.044385            1.0
+
+    Note
+    ----
+    This function tries to compute R_c from equation 16 from
+    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8397498/#FD16. The code is
+    modified based on Shujie Fan's (@VOD555) work.
+
+
+    .. versionadded:: 1.0.0
+    '''
+    series = to_kcalmol(series)
+    array = series.to_numpy()
+    out_length = int(1 / precision)
+    g_forward = _cummean(array, out_length)
+    g_backward = _cummean(array[::-1], out_length)
+    length = len(g_forward)
+
+    convergence = pd.DataFrame(
+        {'Forward': g_forward,
+         'Backward': g_backward,
+         'data_fraction': [i / length for i in range(1, length + 1)]})
+    convergence.attrs = series.attrs
+
+    # Final value
+    g = g_forward[-1]
+    conv = ((g_forward>g-tol) & (g_forward<g+tol)) & ((g_backward>g-tol)&(g_backward<g+tol))
+    for i in range(out_length):
+        if all(conv[i:]):
+            return i / length, convergence
+    else:
+        # This branch exists as we are truncating the dataset to speed the
+        # calculation up. For example if the dataset has 21 points and the
+        # precision is 0.05. The last point of g_forward is computed using
+        # data[0:20] while the last point of g_backward is computed using
+        # data[1:21]. Thus, the last point of g_forward and g_backward are not
+        # the same as this branch will be triggered.
+        return 1.0, convergence
+
+def A_c(series_list, precision=0.01, diff=2):
+    '''Generate the ensemble convergence criteria A_c for a set of simulations.
+
+    The input is a list of pandas.Series generated by
+    :func:`~alchemlyb.preprocessing.subsampling.decorrelate_u_nk` or
+    :func:`~alchemlyb.preprocessing.subsampling.decorrelate_dhdl`.
+
+    The output will the float A_c. A_c is a number between 0 and 1 that can be
+    interpreted as the ratio of the total equilibrated simulation time to the
+    whole simulation time for a full set of simulations. A_c = 1 means that all
+    simulation time frames in all windows can be considered equilibrated, while
+    A_c = 0 indicates that nothing is equilibrated.
+
+    Parameters
+    ----------
+    series_list : list
+        A list of pandas.Series energy array.
+    precision : float
+        The precision of the output R_c.
+    diff : float
+        Tolerance of the convergence check in kcal/mol.
+
+    Returns
+    -------
+    float
+        The area under curve for convergence time fraction.
+
+    Note
+    ----
+    This function tries to compute R_c from equation 18 from
```

Comment:
This function computes $R_c$ ...

Reply:
Do you really mean R_c or should it be A_c.

I'd also put a SeeAlso to R_c and state that the R_c are calculated for the whole set using `R_c()`.